dev/fattest.simplicity/test/componenttest/annotation/processor/TestServletProcessorTests.java

```diff
@@ -201,4 +203,42 @@ public void testProcessorCountManyToOne() {
             assertTrue(expectedNames.contains(method.getMethod().getName()));
         }
     }
+
+    public static class MinJavaLevelTestClass {
+        @TestServlets({
+                        @TestServlet(servlet = Java8Servlet.class, minJavaLevel = 8),
+                        @TestServlet(servlet = Java999Servlet.class, minJavaLevel = 999)
+        })
+        public static LibertyServer NOOP;
+
+        @WebServlet("/*")
+        public static class Java8Servlet extends FATServlet {
+            @Test
+            public void test1_1() {}
+
+            @Test
+            public void test1_2() {}
+        }
+
+        @WebServlet("/*")
+        public static class Java999Servlet extends FATServlet {
+            @Test
+            public void test2_1() {}
+
+            @Test
+            public void test2_2() {}
+        }
+    }
+
+    @Test
+    public void testServletMinJavaLevel() {
+        assumeTrue(8 < JavaInfo.JAVA_VERSION && JavaInfo.JAVA_VERSION < 999);
+
+        List<FrameworkMethod> methods = TestServletProcessor.getServletTests(new TestClass(MinJavaLevelTestClass.class));
+        //Order not guarenteed
```

Comment:
```suggestion
        //Order not guaranteed
```

Reply:
Thanks, I'll fix this in a future PR.  Will merge this now since it's causing build breaks.